src/scripts/main.js

```diff
@@ -1 +1,31 @@
 'use strict';
+
+const cardsNumber = document.querySelectorAll('.carousel__item').length;
+const cardsList = document.querySelector('.carousel__list');
+const cardsButtons = document.querySelectorAll('.carousel__btn');
+const cardsDots = document.querySelector('.carousel__dots-wrap').children;
+
+let numberIndex = 1;
+let translateX = 0;
+
+cardsButtons.forEach(button => {
+  button.addEventListener('click', e => {
+    if (e.target.id === 'previous') {
+      if (numberIndex !== 1) {
+        numberIndex--;
+        translateX += 310;
```

Comment:
It's a kind of a magic number. Put it inside a variable or calculate it using some DOM API methods.